quickstarts/kind/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Kubernes Kind Quickstart
+layout: default
+---
+
+# Installing the dependencies
```

Comment:
Maybe we could add a little bit of introduction about Kind? Even providing a comparison with Minikube?

Reply:
I haven't added it simply because there is no such introduction for Minikube or Minishift either, and I just followed the same template. But I can add some ...

Reply:
I would not get into too many details to be honest. Too muhc stuff to maintain which everyone can learn from the linked pages.

Reply:
I hope it's not too much: https://github.com/strimzi/strimzi.github.io/pull/126/commits/7731702c59467c5689f6fc780e98f9f08804565b

Reply:
+1